libraries/cyclestreets-core/src/main/java/net/cyclestreets/api/client/RetrofitApiClient.java

```diff
@@ -0,0 +1,56 @@
+package net.cyclestreets.api.client;
+
+import net.cyclestreets.api.POI;
+import net.cyclestreets.api.client.dto.FeatureCollection;
+
+import java.io.IOException;
+import java.util.List;
+
+import okhttp3.Interceptor;
+import okhttp3.OkHttpClient;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitApiClient {
+
+  private final V1Api v1Api;
+  private final V2Api v2Api;
+
+  public RetrofitApiClient(String v1Host, String v2Host, String apiKey) {
```

Comment:
I would set the order as `v2Host, v1Host, apiKey` or `v2Host, apiKey, v1Host` as V1 is basically deprecated and the least important of the three.

Also, the V1 API does run over HTTPS if you want, and there's no reason not to do that, so please update references from http://www.cyclestreets.net/ to https://www.cyclestreets.net/